OracleLinux/7/README.md

```diff
@@ -1,36 +1,89 @@
 
 ## Prerequisites
-1. Install [Oracle VM VirtualBox](https://www.virtualbox.org/wiki/Downloads)
-2. Install [Vagrant](https://vagrantup.com/)
+
+Read the [prerequisites in the top level README](../../README.md#prerequisites) to set up either Vagrant with either VirtualBox or KVM
 
 ## Getting started
-1. Clone this repository `git clone https://github.com/oracle/vagrant-boxes`
-2. cd vagrant-boxes/OracleLinux/7
-3. Run `vagrant status` to check Vagrantfile status and possible plugin(s) required
-4. Run `vagrant up`
+
+1. Clone this repository `git clone https://github.com/oracle/vagrant-projects`
+1. Change into the `vagrant-projects/OracleLinux/7` directory
+1. Run `vagrant status` to check Vagrantfile status and possible plugin(s) required
+1. Run `vagrant up`
    1. The first time you run this it will provision everything and may take a while. Ensure you have a good internet connection!
-   2. The Vagrant file allows for customization.
-5. SSH into the VM either by using `vagrant ssh`
+   1. The Vagrant file allows for customization.
+1. SSH into the VM either by using `vagrant ssh`
    If required, by Vagrantfile you can also setup ssh port forwarding.
-6. You can shut down the box via the usual `vagrant halt` and the start it up again via `vagrant up`.
+1. You can shut down the VM via the usual `vagrant halt` and the start it up again via `vagrant up`.
 
 ## Optional plugins
 
-When installed, this Vagrantfile will make use of the following third party Vagrant plugins:
+When installed, this Vagrant project will make use of the following third party Vagrant plugins:
+
+- [vagrant-env](https://github.com/gosuri/vagrant-env): loads environment
+variables from .env files;
 - [vagrant-proxyconf](https://github.com/tmatilai/vagrant-proxyconf): set
 proxies in the guest VMs if you need to access the Internet through proxy. See
 plugin documentation for the configuration.
 - [vagrant-reload](https://github.com/aidanns/vagrant-reload): reload the VM
 during provisioning to activate the latest kernel.
 
-To intall Vagrant plugins run:
-```
+To install Vagrant plugins run:
+
+```shell
 vagrant plugin install <name>...
 ```
 
+## Extending the project
+
+This project can easily be extended by running additional scripts during provisioning and optionally expose guest ports on the host.
+
+Environment variables are used to pass the parameters to the Vagrantfile:
+
+- `EXTEND`: comma separated list of extensions.  
+   For each specified extension a script named `<extension>.sh` is automatically run during provisioning.  
+   Scripts must be located in the `scripts` or `scripts.local` directory.
+- `EXPOSE`: comma separated list of ports to expose in the format: `<host port>:<guest port>`
+
+Example: to extend the project with the `lamp` extension and expose the guest port 80 to 8080 on the host:
+
+```shell
+EXTEND=lamp EXPOSE=8080:80 vagrant up
```

Comment:
This won't work on Windows hosts. The Windows version would be:

```batch
set EXTEND=lamp && set EXPOSE=8080:80 && vagrant up
```

Reply:
Good point...

I'll give an example for both for this one, and change the wording for the 2 others...